src/Examples/IExample.h

```diff
@@ -27,9 +28,21 @@ namespace Examples
         virtual void Suspend()= 0;
         virtual void AfterCameraUpdate() { }
         
+        virtual const Eegeo::m33& getCurrentCameraOrientation(){return *(new Eegeo::m33());}
```

Comment:
It seems that this interface has been extended too far with camera state related getters. The root cause is probably that the VR drawing is taking place too high in the call chain? Or that headTransform[] isn't being used to update a CameraState but is also being based into Draw().